homeassistant/components/folder_watcher.py

```diff
@@ -0,0 +1,85 @@
+"""
+Component for monitoring activity on a folder.
+"""
+import os
+import logging
+import voluptuous as vol
+from watchdog.observers import Observer
+from watchdog.events import FileSystemEventHandler
+
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['watchdog']
```

Comment:
I think we're required to pin the requirements versions. For example

```python
REQUIREMENTS = ['watchdog==0.8.3']
```

Reply:
ok